app/src/main/java/com/example/plantsapp/presentation/ui/notification/TaskNotificationManager.kt

```diff
@@ -68,20 +87,31 @@ class TaskNotificationManager @Inject constructor(
         }
     }
 
-    private fun createPendingIntent(): PendingIntent {
-        val intent = Intent(context, MainActivity::class.java)
-        return PendingIntent.getActivity(
+    private fun createCompletePendingIntent(
+        task: Task,
+        notificationId: Int,
+        requestCode: Int
+    ): PendingIntent {
+        val completeIntent = Intent(context, NotificationBroadcastReceiver::class.java)
+            .apply {
```

Comment:
Here you can move creation of this intent into companion obkject of NotificationBroadcastReceiver